compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -283,7 +283,7 @@ impl<K: DepKind> DepGraph<K> {
         key: DepNode<K>,
         cx: Ctxt,
         arg: A,
-        task: fn(Ctxt, A) -> R,
+        task: fn(Ctxt::DepContext, A) -> R,
```

Comment:
This should stay as `Ctxt`, and the callers should be adapted to pass the correct type.